### PR TITLE
logout and email change

### PR DESCRIPTION
**Pull request by AhdabNasir**

### CODE REVIEW

File: test/unit/pages/Logout.test.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  beforeEach(() => {
+    const Logout = require('pages/logout/Logout');
+
+    res = { redirect: sinon.stub(), clearCookie: sinon.stub() };
+
+    const req = { cookies: [] };
+
+    page = new Logout('some path', req, res);
+
+    sinon.stub(jwt, 'restoreData');
```
Comment (dtaylor7) [inline test/unit/pages/Logout.test.js]: dont think we need this stub?

File: common/views/template.njk [hunk 2]
```diff
@@ -54,6 +61,20 @@
       <a href="/" class="govuk-header__link govuk-header__link--service-name">
         Transition Delivery Dashboard
       </a>
+
+  {% if logout == "true" %}
```
Comment (dtaylor7) [inline common/views/template.njk]: this could be `if req.user`?

Comment (AhdabNasir) [reply common/views/template.njk]: I agree, that'd be a much better way of doing it, or checking if the session is active rather than by page.

Comment (AhdabNasir) [reply common/views/template.njk]: I've changed the code slightly similar to what you were saying here

**Pull request merged**